common-service/src/main/java/org/sunbird/controllers/mapper/RequestMapper.java

```diff
@@ -0,0 +1,39 @@
+/** */
+package org.sunbird.controllers.mapper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.responsecode.ResponseCode;
+import play.libs.Json;
+
+/**
+ * This class will map the requested json data into custom class.
+ *
+ * @author Manzarul
+ */
+public class RequestMapper {
+
+  /**
+   * Method to map request
+   *
+   * @param requestData JsonNode
```

Comment:
requestData Request body JSON data
obj Map to request type
Return mapped request